models/models.js

```diff
@@ -0,0 +1,13 @@
+const { Pool } = require("pg");
+const db = require("../db/connection");
+
+exports.getCategoriesM = (req, res, next) => {
+  return db
+    .query(`SELECT * FROM categories;`)
+    .then(({ rows: categories }) => {
+      return categories;
+    })
+    .catch((err) => {
```

Comment:
be careful adding catches in more than one place in your promise chain, also this is a model not a controller so shouldn't have access to req, res and next. I would only have a catch in the controller